src/GitHub.VisualStudio/UI/Views/Controls/RepositoryCloneControl.xaml

```diff
@@ -1,226 +1,234 @@
 ﻿<uirx:SimpleViewUserControl x:Class="GitHub.VisualStudio.UI.Views.Controls.RepositoryCloneControl"
```

Comment:
The whitespace and alignment was all screwed up on this file, so I reformatted it, hence all the apparent changes :/